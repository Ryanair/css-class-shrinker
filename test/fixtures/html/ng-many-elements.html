<div ng-class="{'aaa-ooo': false}"></div>
<div class="eee">
  <h1 ng-class="{ 'iii': true }"></h1>
</div>
