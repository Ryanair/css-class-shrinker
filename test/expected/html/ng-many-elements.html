<div ng-class="{'dFW': false}"></div>
<div class="eee">
  <h1 ng-class="{ 'fsG': true }"></h1>
</div>
